src/sentry/tasks/auto_remove_inbox.py

```diff
@@ -5,6 +5,6 @@
 from sentry.tasks.base import instrumented_task
 
 
-@instrumented_task(name="sentry.tasks.auto_remove_inbox", time_limit=30, soft_time_limit=20)
+@instrumented_task(name="sentry.tasks.auto_remove_inbox", time_limit=45, soft_time_limit=30)
```

Comment:
Let's just move the ceiling quite a bit higher, back to around what it was before, and let's not worry about it.

I'd say put `time_limit=120, soft_time_limit=110`.